doc-requirements.txt

```diff
@@ -1,25 +1,29 @@
 #
-# This file is autogenerated by pip-compile
+# This file is autogenerated by pip-compile (and then all packages are manually
+# unpinned to use the versions in katsdpdockerbase base-requirements.txt).
+#
 # To update, run:
 #
 #    pip-compile --output-file doc-requirements.txt doc-requirements.in
 #
-alabaster==0.7.10         # via sphinx
-babel==2.5.3              # via sphinx
-certifi==2018.1.18        # via requests
-chardet==3.0.4            # via requests
-docutils==0.14            # via sphinx
-idna==2.6                 # via requests
-imagesize==0.7.1          # via sphinx
-jinja2==2.10              # via sphinx
-markupsafe==1.0           # via jinja2
-pygments==2.2.0           # via sphinx
-pytz==2017.3              # via babel
-requests==2.18.4          # via sphinx
-six==1.11.0               # via sphinx
-snowballstemmer==1.2.1    # via sphinx
-sphinx-rtd-theme==0.2.4
-sphinx==1.6.5
-sphinxcontrib-websupport==1.0.1  # via sphinx
-typing==3.6.4             # via sphinx
-urllib3==1.22             # via requests
+alabaster         # via sphinx
+babel             # via sphinx
+certifi           # via requests
+chardet           # via requests
+docutils          # via sphinx
+idna              # via requests
+imagesize         # via sphinx
+jinja2            # via sphinx
+markupsafe        # via jinja2
+packaging         # via sphinx
+pygments          # via sphinx
+pyparsing         # via packaging
+pytz              # via babel
+requests          # via sphinx
+six               # via packaging, sphinx
+snowballstemmer   # via sphinx
+sphinx-rtd-theme
+sphinx
+sphinxcontrib-websupport    # via sphinx
+typing; python_version<'3'  # via sphinx
+urllib3                     # via requests
```

Comment:
My OCD would be like all the comments to start in the same column.

Reply:
Right. I sort of left them the way pip-compile made them but I see your point - will fix :-)